src/setup/globals.ts

```diff
@@ -71,8 +71,8 @@ export const DEFAULT_IMAGE_SIZES: ImageSizes = {
 /**
  * Contains list of all Pinia stores.
  */
-export enum GlobalStore {
-  SESSION = 'session',
-  BREADCRUMBS = 'breadcrumbs',
-  NOTIFICATION = 'notification',
+export enum Store {
+  Session = 'session',
+  Breadcrumbs = 'breadcrumbs',
```

Comment:
suggestion: in the code we now have a mixture of singular and plural regarding `breadcrumb(s)` and Pinia stores. I would recommend to go with singular (as for e.g. `notification`). I think this is what we once defined for Vuex/Pinia modules.